src/cloud-api/src/error.rs

```diff
@@ -88,4 +88,8 @@ pub enum Error {
     /// Indicates that a timeout has been reached.
     #[error("Timeout reached.")]
     TimeoutError,
+    /// Indicates that the request was executed successfully but returns no content (204).
+    /// E.g.: It happens when the region is enabled but not ready yet.
```

Comment:
this isn't true, we're using the 204 to indicate that the region is not enabled

Reply:
Thank you! I misunderstood. So `204` means it's not enabled, and if it returns `200` and there's no `region_info` does it mean it's enabled but not ready yet?

Reply:
sorta -- these are now the responses:

200 - will include `region_info` and fully activated
202 - enabled but not yet fully active - `region_info` will be `null`
204 - not enabled, no response body at all

basically we just got rid of the 404 since it was causing console errors for the web console and made that return a 204 instead

Reply:
also defined here: https://github.com/MaterializeInc/cloud/blob/main/api/region-api-server.yml#L18

Reply:
Cool. I was browsing in the old definition. 🤦